clang/test/CodeGenCoroutines/coro-await-suspend-destroy-errors.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %clang_cc1 -std=c++20 -verify %s 
```

Comment:
This test should belong to `clang/test/SemaCXX` as this doesn't verify the generated code.